src/main/webapp/app/exercises/programming/hestia/git-diff-report/git-diff-report.component.scss

```diff
@@ -0,0 +1,8 @@
+.diff-placeholder {
```

Comment:
This could be done complety using bootstrap styling in the html `class="d-flex h-100 w-100 justify-content-center align-items-center fs-4"`

Reply:
Good point! I'll remove the custom CSS 👍